### PR TITLE
Add French and German automatic translations for each module (#15)

### PR DESCRIPTION
**Pull request by olbat**
Automatic translations were generated using the raidcore-translator script
(http://github.com/olbat/ws-raidcore-translator).

### CODE REVIEW

**Comment by olbat** [comment]
French and German dumps has been updated in the script's [repository](https://github.com/olbat/ws-raidcore-translator). This modifications has been taken in account in this pull request.

Just for you to know, here is how to use this dumps to rewrite the modules:

```
$ git submodule init
$ git submodule update
$ docker run -v $(pwd):/src/ olbat/ws-raidcore-translator \
    raidcore-translator i18n_v2 -v -l de -t /src/german.json /src/Raidcore/Modules
$ docker run -v $(pwd):/src/ olbat/ws-raidcore-translator \
    raidcore-translator i18n_v2 -v -l fr -t /src/french.json /src/Raidcore/Modules
```

_Note_: Raidcore's [repository](https://github.com/NielsH/RaidCore/tree/master) has been added as a submodule of the project

File: Modules/DSFireWing.lua [hunk 1]
```diff
@@ -37,19 +37,48 @@ mod:RegisterFrenchLocale({
 	["Warmonger Agratha"] = "Guerroyeuse Agratha",
 	["Warmonger Talarii"] = "Guerroyeuse Talarii",
 	["Warmonger Chuna"] = "Guerroyeuse Chuna",
+	["Grand Warmonger Tar'gresh"] = "Grand guerroyeur Tar'gresh",
 	["Conjured Fire Bomb"] = "Bombe incendiaire invoquée",
-	["Totem's Fire"] = "Totem de feu invoqué",
```
Comment (Immemor) [inline Modules/DSFireWing.lua:41]: Could we keep this line.

**Comment by olbat** [comment]
This entry was removed because there is no English version of it (see English i18n array).
It may not be used anywhere or it is a bug in the English version ...

**Comment by Immemor** [comment]
It's right, I added it for the future. I haven't the English entry.

**Comment by olbat** [comment]
Ok, I've made the modification.

**Comment by Immemor** [comment]
Ok, for me, it's perfect. I have nothing to say.

NielsH (or Kami), are you agree ? if yes, I will rebase and push to master.

**Comment by Immemor** [comment]
olbat, can u have a look on https://github.com/NielsH/RaidCore/issues/46.

Thank

**Comment by Immemor** [comment]
I received the Ok by wisp from NielsH.

Rebase done, it's in master.

**Pull request closed without merging**